### ISSUE TITLE
title 'NARA BIA Photography' used on all photo pages

### ISSUE THREAD

**Issue by lfrancis** [usnationalarchives/nara-bia-photos#131]
Some pages have the same title, so the title cannot be used to distinguish pages.	WCAG 2.0 A F25 Section 508 (2017) A F25	378 pages

Change the title elements so they are unique for each page.	
	
'NARA BIA Photography' is also used on https://nara-bia-photos.herokuapp.com/.
https://nara-bia-photos.herokuapp.com/4-h-members-in-garden-285320	Line 18 206	
'NARA BIA Photography' is also used on https://nara-bia-photos.herokuapp.com/.
https://nara-bia-photos.herokuapp.com/a-native-dye-blanket-southern-navajo-agency-1933-298592	Line 18 218	
'NARA BIA Photography' is also used on https://nara-bia-photos.herokuapp.com/.
https://nara-bia-photos.herokuapp.com/bark-house-summer-home-ponemah-point-285713	Line 18 206	
'NARA BIA Photography' is also used on https://nara-bia-photos.herokuapp.com/.
https://nara-bia-photos.herokuapp.com/black-beaver-delawares-519123	Line 18 240	
'NARA BIA Photography' is also used on https://nara-bia-photos.herokuapp.com/.
https://nara-bia-photos.herokuapp.com/buffalo-herd-12463037	Line 18 198	
'NARA BIA Photography' is also used on https://nara-bia-photos.herokuapp.com/.
https://nara-bia-photos.herokuapp.com/campo-reservation-dam-298623	Line 18 214	
'NARA BIA Photography' is also used on https://nara-bia-photos.herokuapp.com/.
https://nara-bia-photos.herokuapp.com/chuck-wagon-285328	Line 18 206	
'NARA BIA Photography' is also used on https://nara-bia-photos.herokuapp.com/.
https://nara-bia-photos.herokuapp.com/corduroy-work-on-the-farm-station-truck-trail-285711	Line 18 206	
'NARA BIA Photography' is also used on https://nara-bia-photos.herokuapp.com/.
https://nara-bia-photos.herokuapp.com/eastern-navajo-agency-pictoral-record-of-fairs-held-fall-1932-298580	Line 18 338

etc etc etc
